misc/git/hooks/govet

```diff
@@ -24,25 +24,14 @@ if [ -z "$GOPATH" ]; then
 fi
 
 # This script does not handle file names that contain spaces.
-gofiles=$(git diff --cached --name-only --diff-filter=ACM | grep '^go/.*\.go$')
+packages=$(git diff --cached --name-only --diff-filter=ACM | grep '^go/.*\.go$' | sed 's|/[^/]*$|/...|' | uniq | sed 's|^|./|')
 
 # If any checks are found to be useless, they can be disabled here.
-# See the output of "go tool vet" for a list of flags.
-vetflags="-all=true"
+# See the output of "go tool vet help" for a list of flags.
```

Comment:
`go tool vet help` doesn't work with go 1.11.4 on ubuntu. I had to use `go doc cmd/vet` to get a list of available checks